person.rb

```diff
@@ -1,13 +1,14 @@
 class Person < Nameable
-  attr_reader :id
-  attr_accessor :name, age
+  attr_reader :id, :rentals
+  attr_accessor :name, :age
 
   def initialize(age, name = 'Unknown', parent_permission: true)
     super()
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
```

Comment:
- [ ]  A person has many rentals :point_down:   
- A method for adding rentals should be present. You can check [this link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) for more info.
```
  def add_rental(book, date)
    # *The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association
    Rental.new(date, book, self)
  end

```